### PR TITLE
👽️(backends) fix double quoting in ClickHouse backend server parameters

### PR DESCRIPTION
**Pull request by SergioSim**
## Purpose

In the current `clickhouse-connect` release (0.5.19) (see [CHANGELOG](https://github.com/ClickHouse/clickhouse-connect/blob/main/CHANGELOG.md#bug-fixes)), the quoting and escaping of array literals in server parameters is handled internally. Thus, we do not need to add quotes on our side.

## Proposal

- [x] remove quoting of `ids` in ClickHouse backend

### CODE REVIEW

File: tests/models/test_converter.py [hunk 1]
```diff
@@ -372,7 +372,7 @@ def test_converter_convert_with_invalid_page_close_event_raises_an_exception(
             list(result)
 
 
-@settings(suppress_health_check=(HealthCheck.function_scoped_fixture,))
+@settings(deadline=None, suppress_health_check=(HealthCheck.function_scoped_fixture,))
```
Comment (quitterie-lcs) [inline tests/models/test_converter.py:375]: Why did you remove this ? Useless?

Comment (SergioSim) [reply tests/models/test_converter.py:375]: I got the following error during the first run in the CI:

```
Unreliable test timings! On an initial run, this test took 308.32ms, which exceeded the deadline of 200.00ms, but on a subsequent run it took 2.27 ms, which did not. If you expect this sort of variability in your test timings, consider turning deadlines off for this test by setting deadline=None.

You can reproduce this example by temporarily adding @reproduce_failure('6.70.2', b'AXicY2BkYAAiKGBkQObBmIzIHHSAQxgTAAAC6QAJ') as a decorator on your test case
============ 1 failed, 775 passed, 55 warnings in 145.33s (0:02:25) ============
```

I suspect the "suspicious" speedup flagged by `hypothesis` is due to caching performed in our `custom_given` fixture. However, I'm not 100% sure; unfortunately, this error only occurs sometimes.

**Review by wilbrdt — APPROVED** [review]

**Review by jmaupetit — APPROVED** [review]
Don't forget to fill the CHANGELOG!

**Comment by SergioSim** [comment]
> Don't forget to fill the CHANGELOG!

Well spotted, thanks) updated the CHANGELOG)
